app.js

```diff
@@ -69,9 +69,9 @@ let App = create({
         let tldr = []
         Object.keys(json.ConnectionReports).forEach((ip) => {
           let report = json.ConnectionReports[ip]
-          if (!report.ValidCertificates) {
+          if (!report.Checks.ValidCertificates) {
             tldr.push(<div className="warning" key={`cert-${tldr.length}`}>
-              WARN: Self-signed cert found for {ip}, this will need to be replaced in the future <a href="https://github.com/matrix-org/matrix-doc/pull/1711">MSC1711</a>
+              WARN: Couldn't find a valid certificate for {ip}, this will need to be fixed in the future as per <a href="https://github.com/matrix-org/matrix-doc/pull/1711">MSC1711</a>
```

Comment:
Perhaps change this to something like:

```suggestion
              WARN: Could not find a valid certificate for {ip}. This will break federation in the future as per <a href="https://github.com/matrix-org/matrix-doc/pull/1711">MSC1711</a>. <a href="https://github.com/matrix-org/synapse/blob/develop/INSTALL.md#tls-certificates">Fix now</a>
```